tests/DynamoDbGlobalScopeTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace BaoPham\DynamoDb\Tests;
+
+use Carbon\Carbon;
+use BaoPham\DynamoDb\DynamoDbQueryBuilder;
+
+/**
+ * Class DynamoDbTimestampTest
```

Comment:
Since we implicitly use the scope trait, might as well allow this property to inherit from the parent Model class. So, not having it here should still work.